packages/contract/README.md

```diff
@@ -10,51 +10,58 @@ $ npm install @truffle/contract
 
 ### Features
 
-* Synchronized transactions for better control flow (i.e., transactions won't finish until you're guaranteed they've been mined).
-* Promises. No more callback hell. Works well with `ES6` and `async/await`.
-* Default values for transactions, like `from` address or `gas`.
-* Returning logs, transaction receipt and transaction hash of every synchronized transaction.
+- Synchronized transactions for better control flow (i.e., transactions won't finish until you're guaranteed they've been mined).
+- Promises. No more callback hell. Works well with `ES6` and `async/await`.
+- Default values for transactions, like `from` address or `gas`.
+- Returning logs, transaction receipt and transaction hash of every synchronized transaction.
 
 ### Usage
 
-First, set up a new web3 provider instance and initialize your contract, then `require("@truffle/contract")`. The input to the `contract` function is a JSON blob defined by [@truffle/contract-schema](https://github.com/trufflesuite/truffle/tree/master/packages/contract-schema). This JSON blob is structured in a way that can be passed to all truffle-related projects.
+First, set up a new web3 provider instance and initialize your contract, then `require("@truffle/contract")`.
+The contract-schema matches the output of the [@truffle/artifactor](https://github.com/trufflesuite/truffle/tree/master/packages/artifactor), so you can pass in the artifact file truffle produces when you run `truffle compile`.
```

Comment:
This still seems kind of unclear to me. Is contract-schema a thing I don't know about? Shouldn't we be talking about the input to the @truffle/contract default export?

Reply:
Ahh I see you moved the other stuff down below. But is contract-schema a thing?